koku/api/settings/tags/mapping/view.py

```diff
@@ -35,6 +34,15 @@
 from reporting.provider.all.models import TagMapping
 
 
+def parse_list(string_to_parse):
+    """Parse a list to a string."""
+    if "[" not in string_to_parse:
```

Comment:
It's a good idea to start with input validation.
```suggestion
    if not isinstance(string_to_parse, str):
        raise TypeError(f"Expected str but got {type(string_to_parse)}")
```

Would it be better to check that the string starts with a left bracket? A contains check seems overly greedy.